frontend/components/molecules/AccountSettings/ProfileSummary.js

```diff
@@ -60,14 +60,11 @@ const AvatarWrapperComponent = ({ children }) => {
 const ProfileSummary = ({dataLoaded = false, 
```

Comment:
Should personal email, phone number and Student ID be displayed here too?

Reply:
Great suggestion. Let's handle that later :) 
https://app.clickup.com/t/2mra9va